docs/pages/deploy/build-project.mdx

```diff
@@ -12,11 +12,13 @@ import { BuildIcon, EasSubmitIcon, EasMetadataIcon, BookOpen02Icon } from '@expo
 
 Whether you have built a native app binary using [EAS](/build/setup/) or [locally](/guides/local-app-development/), the next step in your app development journey is to submit your app to the stores. To do so, you need to create a **production build**.
 
-## Production builds
+## What is a production build
 
```

Comment:
i think we can delete the intro paragraph and "What is a production build" heading and jump right into the "Production builds are submitted to ..." paragraph